mica/archive/aca_l0.py

```diff
@@ -42,6 +42,25 @@
             'arc5gl_query': 'ACA0',
             'fileglob': '*fits.gz'}
 
+ACA_DTYPE_2 = (('TIME', '>f8'), ('QUALITY', '>i4'), ('MJF', '>i4'),
```

Comment:
Change to 
```
ACA_DTYPE_8x8 = tuple((dt[0], dt[1], (8, 8)) if dt[0] == 'IMGRAW' else dt 
                      for dt in ACA_DTYPE)
```
(assuming the only change is in the IMGRAW dtype.

Reply:
yes, only IMGRAW changed.